tests/e2e/vsan_stretched_cluster_utils.go

```diff
@@ -802,3 +802,30 @@ func scaleUpStsAndVerifyPodMetadata(ctx context.Context, client clientset.Interf
 		}
 	}
 }
+
+// hostFailure causes a host in either site to be powered on or off
+func hostFailure(primarySite bool, hostDown bool) {
+	esxHost := fds.secondarySiteHosts
+	if primarySite {
+		esxHost = fds.primarySiteHosts
+	}
+	host := []string{esxHost[0]}
```

Comment:
We should not pick the same host always IMO.
Just accept the host which needs to be powered off/on as a param to this method and pick a random host in the test case and pass it here.